### PR TITLE
Fix/logo

### PR DESCRIPTION
**Pull request by sericaia**
Adds logo default values

```
* dimensions: 135x25 px
* default color: --dash-green-00
```

The reasoning to add this is that it (1) does not make much sense to add everywhere this `color`, `width`, `height` values if they are the de facto values and (2) this way we also prevent errors. 

We also guarantee that it can be changed if really necessary with new props.

### CODE REVIEW

File: src/components/logo/Logo.tsx [hunk 1]
```diff
@@ -1,11 +1,18 @@
 import * as React from 'react';
 import { SvgWrapper } from '@ions/icon/wrapper/SvgWrapper';
 import { IconProps } from '@ions/icon/types';
+import { dashGreen00 } from '@colors';
+
+const defaultProps = {
```
Comment (maryiam) [inline src/components/logo/Logo.tsx:6]: Why don't you set the default values while destructing the props in `Logo` Component?

Comment (sericaia) [reply src/components/logo/Logo.tsx:6]: it seemed more complex and less readable

Comment (pducolin) [reply src/components/logo/Logo.tsx:6]: I would have set default values the destructured props too, but that's personal taste

**Comment by sericaia** [comment]
I noticed something else which is we have actually 3 logos: Small icon (favicon), normal icon (the one added here) and business icon.

Should this Logo component have them all with different defaults for all of them and a 
`type = small | normal | business` or should we have completely different components for them?

Either is fine for me.

**Comment by pducolin** [comment]
> I noticed something else which is we have actually 3 logos: Small icon (favicon), normal icon (the one added here) and business icon.
> 
> Should this Logo component have them all with different defaults for all of them and a
> `type = small | normal | business` or should we have completely different components for them?
> 
> Either is fine for me.
> 
> @pducolin @maryiam @fmatias-dashlane @ni0ki

yes yes yes!!

**Pull request merged**